server/service/integration_mdm_test.go

```diff
@@ -5198,6 +5199,140 @@ func (s *integrationMDMTestSuite) TestInvalidDiscoveryRequest() {
 
 	// Checking if SOAP response contains a valid SoapFault message
 	resSoapMsg := string(resBytes)
+	require.True(s.T(), s.isXMLTagPresent("s:fault", resSoapMsg))
+	require.True(s.T(), s.isXMLTagContentPresent("s:value", resSoapMsg))
+	require.True(s.T(), s.isXMLTagContentPresent("s:text", resSoapMsg))
+}
+
+func (s *integrationMDMTestSuite) TestValidGetPoliciesRequest() {
+	appConf, err := s.ds.AppConfig(context.Background())
+	require.NoError(s.T(), err)
+	appConf.MDM.WindowsEnabledAndConfigured = true
```

Comment:
I wonder if we should have it set to true by default for those integration MDM tests, and just set it to false explicitly at the start of `TestAppConfigWindowsMDM`. In general, tests to check behaviour when MDM is not enabled would be in core integration tests or enterprise integration tests, the `TestAppConfigWindowsMDM` is the exception as it tests switching it on.

Reply:
Good point! I've just moved the logic to SetupSuite() so `WindowsEnabledAndConfigured` gets set there once